seeds/meetingInfo.js

```diff
@@ -0,0 +1,16 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('meetings').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('meetings').insert([
+        {id: 1, meeting_name: 'EDA Staff', attendees: 6, cost: 12.0},
```

Comment:
These seeded meetings don't have timestamps, which may cause issues when trying to order them by when they occured in the history view